ifrs17-template/Test/Tests.ipynb

```diff
@@ -137,7 +164,7 @@
         {
             "cell_type": "code",
             "source": [
-                "#!eval-notebook \"PracticalCase-CSMSwitchTest\""
+                "DataSource.Reset();"
```

Comment:
why is Reset needed? 
Can't each test initialize a disposable workspace from DataSource with Disable option for some types? (RawVariable, IfrsVariable)?

Reply:
As we discussed today, not all the the imports can work on the workspaces, therefore certain cleanups are desirable.  However, to keep in mind the issue, I put this issue on the board, in the backlog, because conceptually it would be nice at least to have an option to run certain work on the workspace. You can turn this draft into issue of you agree. Otherwise, just disregard.  

https://github.com/orgs/Systemorph/projects/3/views/1?pane=issue&itemId=43912883